pkg/volume/csi/csi_util.go

```diff
@@ -121,3 +121,36 @@ func getVolumeDeviceDataDir(specVolID string, host volume.VolumeHost) string {
 	sanitizedSpecVolID := kstrings.EscapeQualifiedNameForDisk(specVolID)
 	return path.Join(host.GetVolumeDevicePluginDir(csiPluginName), sanitizedSpecVolID, "data")
 }
+
+// hasReadWriteOnce returns true if modes contains v1.ReadWriteOnce
+func hasReadWriteOnce(modes []api.PersistentVolumeAccessMode) bool {
+	if modes == nil {
+		return false
+	}
+	for _, mode := range modes {
+		if mode == api.ReadWriteOnce {
+			return true
+		}
+	}
+	return false
+}
+
+// hasReadOnlyMany returns true if modes contains v1.ReadOnlyMany
+func hasReadOnlyMany(modes []api.PersistentVolumeAccessMode) bool {
+	for _, mode := range modes {
+		if mode == api.ReadOnlyMany {
+			return true
+		}
+	}
+	return false
+}
+
+// hadReadWriteMany returns true if modes contains v1.ReadWriteMany
+func hasReadWriteMany(modes []api.PersistentVolumeAccessMode) bool {
```

Comment:
Who uses these other 2 functions? If no one, may be we can add them as we need them.

Reply:
Agreed. Will remove.